corehq/apps/hqcase/export.py

```diff
@@ -57,10 +56,7 @@ def get_matching_owner(case):
             return get_owner_id(case)
 
     def might_be_relevant(case):
-        if filter_group:
-            return get_owner_id(case) in owner_ids
-        else:
-            return not users or get_matching_owner(case)
+        return not filter_group or get_owner_id(case) in owner_ids
```

Comment:
That's right. Previous logic returned True if owners (mobile users) had not been specified (i.e. they were unknown), or, if they had been specified, then only if the case was owned. This contradicted the requirement for [FB 166197](http://manage.dimagi.com/default.asp?166197), where this should also return True for cases not owned by users. 

i.e. `not users or get_matching_owner(case) or not get_matching_owner(case)` == `not users or True` == `True`

If you think I've misinterpreted the bug, please let me know, but I'm pretty certain this is what @amsagoff meant.